src/main/resources/templates/Index.html

```diff
@@ -1,85 +1,87 @@
 <!DOCTYPE html>
-<html xml:th="http://www.thymeleaf.org">
+<html xmlns:th="http://www.thymeleaf.org" lang="en-US">
 <head>
-  <link rel="stylesheet" href="index.css">
+  <title>Voluntier</title>
+  <link rel="stylesheet" th:href="@{css/Index.css}">
+
   <script>
     function openForm() {
       document.getElementById("myForm1").style.display = "block";
     }
 
-    function closeForm() {
-      document.getElementById("myForm").style.display = "none";
+    function closeForm1() {
+      document.getElementById("myForm1").style.display = "none";
+    }
+
+    function closeForm2() {
+        document.getElementById("myForm2").style.display = "none";
     }
   </script>
 </head>
 <body>
 <div class="content">
   <header>
-    <img src="images.png" alt="Logo">
-
+    <img th:src="@{/images/images.png}" alt="Logo">
   </header>
+
   <div class="tab">
-    <button><a href="index.html">About this App</a></button>
+    <button><a href="/">About this App</a></button>
     <button><a href="#">Account</a></button>
     <button><a href="#">Password</a></button>
-    
-    <button><a href="help.html">Help</a></button>
-    <button><a href="#">Apply</a></button>
-    <button><a href="post.html">Create/Post</a></button>
 
+    <button><a href="help">Help</a></button>
+    <button><a href="#">Apply</a></button>
+    <button><a href="post">Create/Post</a></button>
   </div>
 
   <div class="login">
     <button class="open-button" onclick="openForm()">Log In</button>
-
-
-    <p th:text="${volunteer}">
-
-
+    <p th:text="${volunteer}"></p>
 
     <div class="form-popup" id="myForm1">
-      <form action="#" th:action="@{/saveVolunteer}" th:object="${volunteer}"method="GET" class="form-container">
+      <form action="#" th:action="@{/saveVolunteer}" th:object="${volunteer}" method="GET" class="form-container">
 
-        <label for="email"><b>Email</b></label>
-        <input type="text" placeholder="Enter Email" name="email" th:field="*{volunteerEmail}" required>
+        <label for="email1"><b>Email</b></label>
+        <input type="text" placeholder="Enter Email" name="email1" id="email1" th:field="*{volunteerEmail}" required>
 
-        <label for="psw"><b>Password</b></label>
-        <input type="password" placeholder="Enter Password" name="psw" th:field="*{volunteerPass}" required>
+        <label for="psw1"><b>Password</b></label>
+        <input type="password" placeholder="Enter Password" name="psw1" id="psw1" th:field="*{volunteerPass}" required>
```

Comment:
Properly defined `for` attribute for `<label>` by defining `id` attribute for `input` tags being labeled. [Reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)